features/step_definitions/step_defs.js

```diff
@@ -142,7 +142,25 @@ const columnNameMap = new Map([
     ['aanduiding verblijfstitel (39.10)', 'verblijfstitel_aand'],
     ['datum einde verblijfstitel (39.20)', 'verblijfstitel_eind_datum'],
     ['datum ingang verblijfstitel (39.30)', 'geldigheid_start_datum'],
-
+	
+    ['aktenummer (81.20)', 'akte_nr' ],
+	
+    ['gemeente document (82.10)', 'doc_gemeente_code' ],
```

Comment:
een aantal van deze velden worden ook in #1229 (update kinderen feature) toegevoegd. Gaat hopelijk merge conflict geven